packages/xrpl/test/wallet/signer.test.ts

```diff
@@ -139,7 +139,9 @@ describe('Signer', function () {
   it('multisign runs successfully with tx_blobs', function () {
     const transactions = [multisignTxToCombine1, multisignTxToCombine2]
 
-    const encodedTransactions: string[] = transactions.map(encode)
+    const encodedTransactions: string[] = transactions.map((transaction) =>
```

Comment:
Why was this change made?

Reply:
This change was made because the function signature changed, so the implicit syntax doesn't work anymore. (Before it was a single parameter, single output, but since there's now an additional optional parameter it needs an explicit call)